optaplanner-quarkus-integration/optaplanner-quarkus/deployment/src/main/java/org/optaplanner/quarkus/deployment/GizmoMemberAccessorEntityEnhancer.java

```diff
@@ -294,8 +296,37 @@ public static String generateSolutionCloner(SolutionDescriptor solutionDescripto
                         transformers);
             }
 
+            DeepCloningUtils deepCloningUtils = new DeepCloningUtils(solutionDescriptor);
+
+            Set<Class<?>> solutionAndEntitySubclassSet = new HashSet<>(solutionSubclasses);
```

Comment:
According to the convention, `solutionSubclasses` should be a `solutionSubclassesList`.
Also, not sure why the copy is being made - maybe I missed something in the diff, but it seems like you don't need the original collection after this.

Reply:
Original list is used on 328 (and expects the list to contain only solution subclasses)